site2/website/coding-guide.md

```diff
@@ -20,7 +20,7 @@ Apache Pulsar uses the following libraries a lot:
 
 Use these libraries whenever possible rather than introducing more dependencies. 
 
-Dependencies are bundled with our binary distributions, so we need to attach the relevant licenses. See [Third party dependencies and licensing](/community/licensing) for a guide on how to do it correctly.
+Dependencies are bundled with our binary distributions, so we need to attach the relevant licenses. See [Third party dependencies and licensing](https://pulsar.apache.org/docs/en/client-libraries/) for a guide on how to do it correctly.
```

Comment:
@Jennifer88huang I went through Pulsar docs, and thought this could be the one. If there are any changes, please do let me know, I'll be glad to fix it up :)

Reply:
Don't worry about the CI check. After approval, we will run the tests.
I'll check the link and let you know when I have the answer.

Reply:
Okay Sure, Thanks a lot 🚀